pkg/config/config.go

```diff
@@ -141,6 +141,12 @@ func init() {
 	Datadog.SetDefault("kubernetes_http_kubelet_port", 10255)
 	Datadog.SetDefault("kubernetes_https_kubelet_port", 10250)
 
+	Datadog.SetDefault("kubelet_tls_verify", true)
+	Datadog.SetDefault("kubelet_auth_token_path", "")
+	Datadog.SetDefault("kubelet_client_crt", "")
+	Datadog.SetDefault("kubelet_client_key", "")
+	Datadog.SetDefault("kubelet_client_ca", "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt")
```

Comment:
we should probably `BindEnv` these?

Reply:
yes good idea

Reply:
As these options are only intended for use when running on the host (we should use the pod's serviceaccount when running in a pod), it's less high priority than other options though

Reply:
@xvello the two options for serviceaccount are:
* kubelet_client_ca: already set to serviceAccount
* kubelet_auth_token_path: unset by default to identify custom [setting](https://github.com/DataDog/datadog-agent/pull/1073/files#diff-d612d3d6decd2d6d70354157c9af19adR16)

Then I still use serviceAccount token in [step 3](https://github.com/DataDog/datadog-agent/pull/1073/files#diff-731ede6c6bb1577c969c582622a44f2dR127).

As alternative I can provide a `const` in the config default **kubelet_auth_token_path** used [here](https://github.com/DataDog/datadog-agent/pull/1073/files#diff-731ede6c6bb1577c969c582622a44f2dR151).
Then it changes this [part](https://github.com/DataDog/datadog-agent/pull/1073/files#diff-d612d3d6decd2d6d70354157c9af19adR17) to: 
```go
config.Datadog.GetString("kubelet_auth_token_path") != kubernetes.ServiceAccountTokenPath
```
But it's ambiguous if the user configure the value of the const that's why I used an empty string.

Reply:
I'd rather both be consistent. My preference goes to having an empty default that the logic detects and replaces by the internal constant. That way we can debug log that we're using the default. The default can still be documented (as a comment) in the config template.

Reply:
Providing a default to **kubelet_client_ca** is not influencing the auto-configuration at all, so I prefer having it like that and not creating a new constant and it's associated function to set it.